src/Atmos/Model/AtmosModel.jl

```diff
@@ -536,11 +536,18 @@ function. Contributions from subcomponents are then assembled (pointwise).
     aux::Vars,
     t::Real,
 )
+    ρu_pad = SVector(1, 1, 1)
+    ts = recover_thermo_state(atmos, state, aux)
+    tend = Flux{SecondOrder}()
+    args = (atmos, state, aux, t, ts, diffusive, hyperdiffusive)
+    flux.ρ = Σfluxes(eq_tends(Mass(), atmos, tend), args...) .* ρu_pad
+    flux.ρu = Σfluxes(eq_tends(Momentum(), atmos, tend), args...) .* ρu_pad
+    flux.ρe = Σfluxes(eq_tends(Energy(), atmos, tend), args...)
```

Comment:
My English translation of this line is:

> Set the energy flux, `flux.ρe`, to a sum of fluxes, `Σfluxes`, which depend on "`Energy` equation tendencies", `(eq_tends(Energy()...)`

The last bit is non-sensical to me though, so I can't quite figure out what the code is telling me. I guess a tendency typically depends on flux divergences in the sense that a generic equation can be written

```
∂t Q = - ∇ ⋅ F + S
```

So fluxes and tendencies are related. But the tendency doesn't go into the flux; instead, the flux divergence is one of the contributions to a quantity's tendency. Maybe I am missing something?

Reply:
I think you're right, `eq_tends` technically returns `TendencyDef{TT}`'s, where `TT <: Union{Flux{AbstractOrder},Source}`, I think part of the naming challenge is that there's not really a good "supertype" name for both a flux, whose tendency is the divergence of the flux, and the source, which is a tendency itself. All that said, this is the way it is because we couldn't come up with better names.